docs/portal/developer-portal/advanced/Enabling_K3s.md

```diff
@@ -134,6 +134,8 @@ metallb_ipaddresspool_range_start: "x.x.x.x"
 metallb_ipaddresspool_range_end: "x.x.x.x"
 ```
 
+MetalLB will assign an IP address to each service running in K3s that requires and external IP address. In the case of HAProxy, each instance of HAProxy will require an IP address. Podman containers do *not* require their own IP address.
```

Comment:
in the future when this feature because more supported and mature, we can use this fact as the start of the decision tree  which will help me procedure-fy this and the following config sections.